request-manager/graph/resolver.go

```diff
@@ -556,22 +535,21 @@ func (o *resolver) buildSingleVertexGraph(nodeDef *spec.Node, jobArgs map[string
 	return g, nil
 }
 
-// NewReqGraph creates an "empty" graph. It contains two nodes: the "start" and "end" nodes. Both of these nodes
-// are no-op jobs
-func (o *resolver) newReqGraph(name string, jobArgs map[string]interface{}) (*Graph, error) {
+// newReqGraph creates an "empty" graph. It contains two nodes: the noop source and sink nodes.
+func (r *resolver) newReqGraph(name string, jobArgs map[string]interface{}) (*Graph, error) {
 	var err error
```

Comment:
This var isn't really used until `err = setNodeArgs(&spec.NoopNode, jobArgs, jobArgsCopy)` (not shown in GitHub diff). Let's remove it and do,
```go
if err := setNodeArgs(&spec.NoopNode, jobArgs, jobArgsCopy); err != nil {
```